specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-04-01/machineLearningServices.json

```diff
@@ -4701,6 +4717,260 @@
+          "format": "int32",
+          "type": "integer",
+          "description": "Application port inside the container."
+        },
+        "published": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Port over which the application is exposed from container.",
+          "x-nullable": true
+        },
+        "hostIp": {
+          "description": "Host IP over which the application is exposed from the container",
+          "type": "string",
+          "x-nullable": true
+        }
+      }
+    },
+    "VolumeDefinition": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "bind",
+            "volume",
+            "tmpfs",
+            "npipe"
+          ],
+          "default": "bind",
+          "title": "Type of Volume Definition",
+          "description": "Type of Volume Definition. Possible Values: bind,volume,tmpfs,npipe",
+          "x-ms-enum": {
+            "name": "VolumeDefinitionType",
+            "modelAsString": true
+          },
+          "type": "string"
+        },
+        "readOnly": {
+          "type": "boolean",
+          "description": "Indicate whether to mount volume as readOnly. Default value for this is false.",
+          "x-nullable": true
+        },
+        "source": {
+          "type": "string",
+          "description": "Source of the mount. For bind mounts this is the host path."
+        },
+        "target": {
+          "type": "string",
+          "description": "Target of the mount. For bind mounts this is the path in the container."
+        },
+        "consistency": {
+          "type": "string",
+          "description": "Consistency of the volume"
+        },
+        "bind": {
+          "$ref": "#/definitions/BindOptions",
+          "description": "Bind Options of the mount"
+        },
+        "volume": {
+          "$ref": "#/definitions/VolumeOptions",
+          "description": "Volume Options of the mount"
+        },
+        "tmpfs": {
+          "$ref": "#/definitions/TmpfsOptions",
+          "description": "tmpfs option of the mount"
+        }
+      }
+    },
+    "VolumeOptions": {
+      "type": "object",
+      "properties": {
+        "nocopy": {
+          "type": "boolean",
+          "description": "Indicate whether volume is nocopy",
+          "x-nullable": true
+        }
+      }
+    },
+    "BindOptions": {
+      "type": "object",
+      "properties": {
+        "propagation": {
+          "type": "string",
+          "description": "Type of Bind Option",
+          "x-nullable": true
+        },
+        "createHostPath": {
+          "type": "boolean",
+          "description": "Indicate whether to create host path.",
+          "x-nullable": true
+        },
+        "selinux": {
+          "type": "string",
+          "description": "Mention the selinux options.",
+          "x-nullable": true
+        }
+      }
+    },
+    "TmpfsOptions": {
```

Comment:
>"TmpfsOptions": { [](http://example.com/codeflow?start=4&length=17)

Please add Title or Description to this definition.

Reply:
Added, Thanks for bringing it up.